src/Network/Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -92,6 +92,12 @@ public class NewAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
             HelpMessage = "EnableBgp Flag")]
         public bool EnableBgp { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Disable IPsec Protection Flag")]
+        public bool DisableIPsecProtection { get; set; }
+
```

Comment:
Please also update the help document.

Reply:
Thank you Wyunchi, I have updated the help document for New-AzVitualNetworkGateway.

Reply:
Greate! Thanks!